src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+  </head>
+  <body>
+    <div class="product_card" data-qa="card">
+      <img src="./images/imac.jpeg"
+           alt="imac"
+           class="card_img"
+      >
+      <div class="card_item">
+         Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card_id">
+         Код товара: 195434
+      </div>
+      <section class="card_review">
+        <div class="card_review_stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+        <div class="card_review_feedback">
+         Отзывов: 5
+        </div>
+      </section>
+      <section class="card_price">
+        <div class="card_price_cost">Цена:</div>
+        <div class="card_price_total">69 999 грн</div>
+      </section>
+      <a href="//"
+         class="card_buy_button"
+         data-qa="hover"
+      >
```

Comment:
Не очень хороший стиль кода
```suggestion
      <a
         href="//"
         class="card_buy_button"
         data-qa="hover"
      >
```